docs/quickstarts/nextjs.mdx

```diff
@@ -214,21 +211,10 @@ export default function Home() {
 
 Then, visit your app's homepage at [`localhost:3000`](http://localhost:3000) to see the `<UserButton />` in action. It should show your avatar from the account you signed in with.
 
-### Sign out
-
-Finally, use the `<UserButton />` by selecting your avatar, then selecting **Sign out**.
-
 </Steps>
 
 ## Next steps
 
-To see an example application that demonstrates user authentication, and session and organization management, explore one of Clerk's Next.js demo repos:
-
-- [Clerk + Next.js App Router Demo](https://github.com/clerk/clerk-nextjs-demo-app-router)

```

Comment:
I don't think we should remove these fully - users use these as demonstrative repositories of our features, and for their code. We can instead do what we did in the Core 2 docs: https://beta.clerk.com/docs/quickstarts/nextjs

<img width="755" alt="Screenshot 2024-03-13 at 12 52 16" src="https://github.com/clerk/clerk-docs/assets/98043211/a7f13358-63ac-4c5a-b248-66a9c9b78f1a">